GVFS/GVFS.Installer.Mac/uninstall_vfsforgit.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+KEXTFILENAME="PrjFSKext.kext"
+VFSFORDIRECTORY="/usr/local/vfsforgit"
+PRJFSKEXTDIRECTORY="/Library/Extensions"
+GVFSCOMMANDPATH="/usr/local/bin/gvfs"
+UNINSTALLERCOMMANDPATH="/usr/local/bin/uninstall_vfsforgit.sh"
+INSTALLERPACKAGEID="com.vfsforgit.pkg"
+KEXTID="io.gvfs.PrjFSKext"
+
+function UnloadKext()
+{
+    kextLoaded=`/usr/sbin/kextstat -b "$KEXTID" | wc -l`
+    if [ $kextLoaded -eq "2" ]; then
+        unloadCmd="sudo /sbin/kextunload -b $KEXTID"
+        echo "$unloadCmd..."
+        eval $unloadCmd || exit 1
+    fi
+}
+
+function UnInstallVFSForGit()
+{
+    if [ -d "${VFSFORDIRECTORY}" ]; then
+        rmCmd="sudo /bin/rm -Rf ${VFSFORDIRECTORY}"
+        echo "$rmCmd..."
+        eval $rmCmd || exit 1
+    fi
+    
+    if [ -d "${PRJFSKEXTDIRECTORY}/$KEXTFILENAME" ]; then
+        rmCmd="sudo /bin/rm -Rf ${PRJFSKEXTDIRECTORY}/$KEXTFILENAME"
+        echo "$rmCmd..."
+        eval $rmCmd || exit 1
+    fi
+    
+    if [ -s "${GVFSCOMMANDPATH}" ]; then
+        rmCmd="sudo /bin/rm -Rf ${GVFSCOMMANDPATH}"
+        echo "$rmCmd..."
+        eval $rmCmd || exit 1
+    fi
+}
+
+function ForgetPackage()
+{
+    if [ -f "/usr/sbin/pkgutil" ]; then
+        forgetCmd="sudo /usr/sbin/pkgutil --forget $INSTALLERPACKAGEID"
```

Comment:
What does this line do?

Reply:
Mac installer keeps a bom (Bill of materials) for each package that it installs in a `receipts` db. You can find them here - `/var/db/receipts`. It contains info on the files packages install. `pkgutil --forget` command removes the receipt.
<img width="737" alt="screen shot 2019-01-16 at 11 12 12 am" src="https://user-images.githubusercontent.com/378580/51262927-4fb13580-1981-11e9-85d1-edb2bd638a75.png">